build/travis-debug/Dockerfile

```diff
@@ -0,0 +1,53 @@
+# Simulate our Maven build on Travis-CI
+# WORKDIR ends up as /home/travis/build
+# Source directory should be mounted here
+
+FROM quay.io/travisci/travis-jvm
```

Comment:
How do we verify that this is the Travis provided image? Is there some sort of signature checking on the binary? quay.io is a third party vendor.